features/async-iterators.yml

```
@@ -0,0 +1,13 @@
+name: Async iterators and the for await..of loop
```

This would make it consistent with the "Array iterators" and "Typed array iterators" feature names. What do you think?

```suggestion
name: Async iterators
```

---

I think the consistency we want to achieve here is with https://github.com/web-platform-dx/web-features/pull/1658 rather than with Array and TypedArray built-ins that gained iterable functionality.

---

So in that case, I would vote for dropping the "Async iterators" part and using "for await..of loop" instead. And then in 1658: "for..of loop".

---

https://chromestatus.com/feature/4696563918045184 also labeled this "Iterators and for-of loop". 

I think it would be good to mention that there are sync iterators and asynchronous iterators with the respective loops. So. yeah either we name these two features:

- "Async iterators and the for await..of loop"
- "Iterators and the for...of loop"

or

- "for await..of loop"
- "for...of loop"

---

* "Async iterators and the for await..of loop"
* "Iterators and the for...of loop"

seem good to me